awx/main/signals.py

```diff
@@ -399,6 +412,9 @@ def activity_stream_create(sender, instance, created, **kwargs):
         else:
             activity_entry.setting = conf_to_dict(instance)
             activity_entry.save()
+        connection.on_commit(
+            lambda: emit_activity_stream_change(activity_entry)
```

Comment:
The only thing that concerns me is that this will now be ran in migrations. If some migration created 1M activity stream entries, these methods would pile up, and that would cause problems. I don't feel like migrations should create _any_ activity stream entries, and I disable them when the issue comes up.